presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -634,7 +634,15 @@ public PlanOptimizers(
                         ruleStats,
                         statsCalculator,
                         estimatedExchangesCostCalculator,
-                        ImmutableSet.of(new RemoveRedundantIdentityProjections())));
+                        ImmutableSet.of(new RemoveRedundantIdentityProjections())),
+                new IterativeOptimizer(
+                        metadata,
+                        ruleStats,
+                        statsCalculator,
+                        estimatedExchangesCostCalculator,
+                        ImmutableSet.<Rule<?>>builder()
+                                .addAll(new InlineSqlFunctions(metadata, sqlParser).rules())
```

Comment:
Do we still need the earlier pass?

Reply:
I wasn't sure at first but I briefly skimmed all the rules between the original and new location and I feel pretty confident it can be moved rather than copied

Reply:
We might need to keep both--if we don't inline SQL functions at the beginning, then we might lose downstream opportunities to optimize them.

Reply:
Agree that keeping the earlier pass is better.